setup.cfg

```diff
@@ -68,7 +68,7 @@ test =
     pytest
     pytest-cov
     tabulate
-    pandas;python_version>"3.4"
+    pandas;python_version>"3.4" and python_version<"3.10"
```

Comment:
Check now takes 54 seconds instead of 18 minutes. :)

Reply:
Cool 👍! (But of course that time gets a lot down after getting up a lot with the on-the-spot builds for 3.10 ;-).)